src/bicep-types/src/types.ts

```diff
@@ -416,5 +416,7 @@ export interface TypeSettings {
   name: string;
   version: string;
   isSingleton: boolean;
+  isPreview?: boolean;
+  isDeprecated?: boolean;
```

Comment:
I think you'll also need to add to https://github.com/Azure/bicep-types/blob/4777894c5f49555628efb061f90b32b26887a52c/src/Bicep.Types/Index/TypeSettings.cs#L9

Reply:
There are also a few tests that it would be good to update - see https://github.com/search?q=repo%3AAzure%2Fbicep-types%20configurationType&type=code for example

Reply:
Good to know!

Reply:
Updated the C# type and added the new settings to the `foo` baseline.